tutorials/evaluation.md

```diff
@@ -9,59 +9,71 @@ You can evaluate LitGPT using [EleutherAI's lm-eval](https://github.com/Eleuther
 You need to install the `lm-eval` framework first:
 
 ```bash
-pip install 'lm_eval @ git+https://github.com/EleutherAI/lm-evaluation-harness.git@115206dc89dad67b8b'
+pip install lm_eval
 ```
 
 &nbsp;
 
 ### Evaluating LitGPT base models
 
-Use the following command to evaluate LitGPT models on all tasks in Eleuther AI's Evaluation Harness.
+Suppose you downloaded a base model that we want to evaluate. Here, we use the `microsoft/phi-2` model:
 
 ```bash
-python eval/lm_eval_harness.py \
-    --checkpoint_dir "checkpoints/meta-llama/Llama-2-7b-hf" \
-    --precision "bf16-true" \
-    --save_filepath "results.json"
+litgpt download --repo_id microsoft/phi-2
 ```
 
-To evaluate on LLMs on specific tasks, for example, TruthfulQA and HellaSwag, you can use the `--eval_task` flag as follows:
+The download command above will save the model to the `checkoints/microsoft/phi-2` directory, which we can
+specify in the following evaluation command:
 
-```bash
-python eval/lm_eval_harness.py \
-    --checkpoint_dir "checkpoints/meta-llama/Llama-2-7b-hf" \
-    --eval_tasks "[truthfulqa_mc,hellaswag]" \
-    --precision "bf16-true" \
-    --save_filepath "results.json"
+
+```
+litgpt evaluate \
+  --checkpoint_dir checkpoints/microsoft/phi-2/ \
+  --out_dir evaluate_model/
 ```
 
-A list of supported tasks can be found [here](https://github.com/EleutherAI/lm-evaluation-harness/blob/master/docs/task_table.md).
+Please note that the `litgpt eval` command run an internal model conversion. 
+This is only necessary the first time you want to evaluate a model. To skip the conversion, 
+when you want to evaluate a model a second time, you can pass the `--skip_conversion true` argument:
+
+```
+litgpt evaluate \
+  --checkpoint_dir checkpoints/microsoft/phi-2/ \
+  --out_dir evaluate_model/ \
+  --skip_conversion true
+```
 
 &nbsp;
 
-### Evaluating LoRA-finetuned LLMs
+> [!TIP]
+> By default, `ligpt evaluate` will evaluate a model on all Open LM Leaderboard tasks, which corresponds
+to the setting `--tasks "hellaswag,gsm8k,truthfulqa_mc2,mmlu,winogrande,arc_challenge"`. 
```

Comment:
Looks like this was a bad splitting via the Note box on GitHub. Annoying, but "hellaswag,gsm8k,truthfulqa_mc2,mmlu,winogrande,arc_challenge" needs to be a on a separate line to avoid this.